app/tests/test_basic.py

```diff
@@ -0,0 +1 @@
+import unittest
```

Comment:
Incomplete

Reply:
@Geek-ubaid 
Yes, I need advice on what tests can be included here.

Reply:
Since you will be testing the routes using postman. Include test examples in the docs so no tests for routes.

Include test for socket connections like broadcast and marking of attendance.

Reply:
https://github.com/GDGVIT/AttendanceApp-Backend/issues/9#issue-559112858